packages/vue/src/DataDisplay/FloatContent/OcFloatContent.vue

```diff
@@ -0,0 +1,132 @@
+<script setup>
+import { CopyTooltip, Dropdown, DropdownItem, Icon } from "@/orchidui";
+import { ref } from "vue";
+
+const props = defineProps({
+  modelValue: {
+    type: Boolean,
+    default: false,
+  },
+  isClose: {
+    type: Boolean,
+    default: true,
+  },
+  isDropdown: {
+    type: Boolean,
+    default: true,
+  },
+  title: String,
+  description: String,
+  dropdownOptions: {
+    type: Object,
+    default: () => ({
+      top: [],
+      bottom: [],
+    }),
+  },
+  dropdownProps: {
+    type: Object,
+    default: () => ({}),
+  },
+  position: {
+    type: String,
+    validator: (v) => ["left", "right"].includes(v),
+    default: "right",
+  },
+});
+
+const emit = defineEmits(["update:model-value"]);
+
+const isOpen = ref(false);
+</script>
+
+<template>
+  <div
+    class="fixed top-0 bottom-0 shadow-xl bg-oc-bg-light transition-all h-full"
+    :class="{
+      'right-0': modelValue && position === 'right',
+      'right-[-999px] w-0': !modelValue && position === 'right',
```

Comment:
Can it be? apply for `position === 'left'` if possible

```suggestion
      'invisible': !modelValue && position === 'right',
```